decidim-core/lib/decidim/data_portability_serializers.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module DataPortabilitySerializers
+    autoload :DataPortabilityUserSerializer, "decidim/data_portability_serializers/data_portability_user_serializer"
```

Comment:
Maybe prepending `DataPortability` to each class inside `DataPortabilitySerializers` is a bit redundant?